.github/workflows/build_artifact.yml

```diff
@@ -0,0 +1,32 @@
+name: Build OpenMRS Distro Bahmni
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Build with Maven
+      run: |
+       ./mvnw -s $GITHUB_WORKSPACE/.github/workflows/settings.xml clean install
+      env:
+        USER_NAME: ${{ secrets.USERNAME }}
+        ACCESS_TOKEN: ${{ secrets.PASSWORD }}
+
+     
+    - name: Upload artifacts 
+      uses: actions/upload-artifact@v2
+      with:
+          name: Bahmnicore
```

Comment:
Can we rename the artifact name to `BahmniDistro` because the artifact includes all other OMODs of Bahmni including bahmni-core ?

Reply:
Renamed to `BahmniDistro`